05-extra.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: page
+title: Version Control with Git
+subtitle: Additional Topics
```

Comment:
What about our templates requires the extras file to be split? I'd prefer
it to stay one file. Is there a way we can make that work?
On Mon, Feb 9, 2015 at 1:41 PM Andy Boughton <notifications@github.com>
wrote:

> In 05-extra.md
> <https://github.com/swcarpentry/git-novice/pull/36#discussion_r24368034>:
>
> > @@ -0,0 +1,225 @@
> > +---
> > +layout: page
> > +title: Version Control with Git
> > +subtitle: Additional Topics
>
> Prepping a pull request as we speak.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swcarpentry/git-novice/pull/36/files#r24368034>.
>